content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+* Run analytics on data with tools like Apache Spark, Apache Drill, Presto, and S3 Select to gain valuable insights into customers, operations, or markets.
+* Maintain Spark Delta Lake time travel information. You can time travel to see different versions of the data when Object Store is configured as a data lake for Spark Delta Lake.
+* Store ML model data and share the ML models in real-time with downstream applications.
+* Publish S3 events to HPE Ezmeral Data Fabric Streams.
+
+## [](#install-and-configure-object-store)Install and configure Object Store
+
+Regarding the installation of Object Store, I recommend you to use the [Installer](https://docs.datafabric.hpe.com/72/MapRInstaller.html).
+
+When using the Installer to install HPE Ezmeral Data Fabric 7.0.0 or higher, you must enable security.
+
+Even for POC environments, I recommend you enable and configure basic security.
+
+If you have ever used the ecological components of Apache Hadoop, or other commercial big data suites, then I think you already have the basic concepts of authentication, authorization, audit, and encryption of the Hadoop ecosystem.
+
+The rationale for the security of HPE Ezmeral Data Fabric is the same as an open-source Hadoop ecosystem.
+
+For example, SASL (MapR-SASL, Kerberos) is used for authentication, Ranger is used for authorization, and TLS is used for encryption.
+
+Use the Installer to install HPE Ezmeral Data Fabric, which can automatically create a series of TLS-related certificates, and automatically configure core components and various HPE Ezmeral Ecosystem Pack components to enable security.
+
+### [](#post-installation-configuration-for-object-store)Post-installation configuration for Object Store
+
+Some post-installation steps must be performed before you can use the HPE Ezmeral Data Fabric Object Store.
+
+You should refer to this document - [Enabling the HPE Ezmeral Data Fabric Object Store](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html) for post-installation configuration.
+
+For the above document, I have a few supplementary notes, which should make your configuration smoother.
+
+#### 1. About the "keytool -noprompt -importcert" command in the docs.
+
+Please do not use this 👇 command in the original document:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias maprca -keystore ${JAVA_HOME}/lib/security/cacerts -storepass <cacerts_truststore>
+```
+
+Instead, use the following command:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias maprca -cacerts -storepass <cacerts_truststore>
+```
+
+If you use the Installer to install HPE Ezmeral Data Fabric on a fresh OS, then Installer will automatically install JDK 11, then "-storepass" password is "changeit".
+
+There is another place in the documentation where keytool is used, the command is as follows:
```

Comment:
I recommend breaking this out into two sentences, placing a period after "...keytool is used." and starting the next sentence with "The command is as follows:"